README.md

```diff
@@ -1,13 +1,10 @@
-# SangeFault
+# <img src="https://raw.githubusercontent.com/SangeloDev/SangeFault/1.8.9/pack.png" width="32"> SangeFault

```

Comment:
I'll do this manually with a custom logo later.

Reply:
Can be replaced then, at least there's something already before that happens.